includes/Modules/Idea_Hub.php

```diff
@@ -40,7 +38,7 @@
  * @ignore
  */
 final class Idea_Hub extends Module
-	implements Module_With_Scopes, Module_With_Settings, Module_With_Debug_Fields, Module_With_Assets {
+	implements Module_With_Scopes, Module_With_Assets {
```

Comment:
We would also need to delete the `Module_With_Settings_Trait` trait usage and `on_deactivation` and `setup_settings` methods if we removed the `Module_With_Settings` interface.

Reply:
Removed in https://github.com/google/site-kit-wp/pull/3557/commits/dec1ebae3fc526df0df1491e17832d2567e3a098